mod/state-transition/pkg/core/helpers_test.go

```diff
@@ -93,7 +153,7 @@ var (
 	testCodec    = &encoding.SSZInterfaceCodec[*types.ExecutionPayloadHeader]{}
 )
 
-func initTestStore() (
+func testInitStore() (
```

Comment:
nit to have all test helpers starting with `test`. Not sure we want that but seems clean to me

Reply:
Same comment here, I feel this would break the existing pattern.